venv/pysnes/cpu.py

```diff
@@ -1792,6 +1865,55 @@ def compute_logic_operation(self, result):
         self.compute_NZflags(flag_result, self.isM())
         return result
 
+    def compute_bit_flags(self, value):
+        if self.isM():
+            if value & 0x80:
+                self.setN()
+            else:
+                self.clearN()
+            if value & 0x40:
+                self.setV()
+            else:
+                self.clearV()
+            if not value & (self.A & 0x00FF):
+                self.setZ()
+            else:
+                self.clearZ()
+        else:
+            if value & 0x8000:
+                self.setN()
+            else:
+                self.clearN()
+            if value & 0x4000:
+                self.setV()
+            else:
+                self.clearV()
+            if not value & self.A:
+                self.setZ()
+            else:
+                self.clearZ()
+
+    def compute_trb_tsb(self, value, isTSB=False):
```

Comment:
Maybe two function are more readable. One for tsb and one for trb

Reply:
I actually had that before and refactored it to be a single function. I'll revert it back to being 2 functions again.

Reply:
Ups sorry, I dont know 😅